ncm-opennebula/src/main/pan/components/opennebula/config.pan

```diff
@@ -5,6 +5,19 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
-include { 'components/${project.artifactId}/sudo' };
+
+include 'components/${project.artifactId}/schema';
+
+bind '/software/components/${project.artifactId}' = component_${project.artifactId};
+
+# Set prefix to root of component configuration.
+prefix '/software/components/${project.artifactId}';
+'active' ?= true;
+'dispatch' ?= true;
+'version' ?= '${no-snapshot-version}';
+'dependencies/pre' ?= list('spma', 'accounts', 'sudo', 'useraccess');
+
+# Package to install
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+include 'components/${project.artifactId}/sudo';
```

Comment:
That was already included in the old `config.pan` file, required to setup oneadmin sudoers conf to be able to execute some ovs commands for example.